.github/workflows/nix-check.yml

```
@@ -0,0 +1,42 @@
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    permissions:
+      contents: read
+      actions: write # needed for purging the cache
+
+    steps:
+      - uses: actions/checkout@de0fac2e4500dabe0009e67214ff5f5447ce83dd
+      - with:
```

actionlint (1.7.12)

[error] step must run script with "run" section or run action with "uses" section

(syntax-check)